tests/utils/testutils.py

```diff
@@ -267,6 +276,18 @@ def tearDownClass(cls):
         cls.webhost = None
 
         if cls.host_stdout is not None:
+            if is_envvar_true(ARCHIVE_WEBHOST_LOGS):
+                version_info = sys.version_info
+                log_file = (
+                    "logs/"
+                    f"{cls.__name__}_"
```

Comment:
De we need to add [`__file__`](https://docs.python.org/3/reference/import.html#file__) option rather than name, to help with the issue I was mentioning? or that is specific to async tests only?

Reply:
from debug console, i dont see teardownclass param has __file__, 
cls.__name__
'TestLogFilteringFunctions'
cls.__file__
Traceback (most recent call last):
  File "<string>", line 1, in <module>
AttributeError: type object 'TestLogFilteringFunctions' has no attribute '__file__'

for these sync tests, the module name is the filename. i can add cls.__module__.